### ISSUE TITLE
Error occured when invoking location through minecraft client

### ISSUE THREAD

**Issue by Vigilans** [LXGaming/Location#3]
`Location` can work in the my velocity server interactive console now (through `location <some_command>`). But when invoking location with typing `/location ...` in a minecraft game client, following error ocurred:
```
<SomePlayer> /location
↓
[Location] Use /location help to view available commands

<SomePlayer> /location <any_command>
↓
[Location] An error has occurred:
Unknown command at position 0: <-- [HERE]
```
Is this a bug or intended?

**Comment by LXGaming** [comment]
What Minecraft Client/Server and Velocity version?

**Comment by Vigilans** [comment]
Minecraft Client/Server: 1.18.2
Velocity: 3.2.0-SNAPSHOT
Location: 1.4.0

Had already been not working in earlier versions:
Minecraft Client/Server: 1.18.1 and 1.18.2
Velocity: 3.1.1-98
Location: 1.3.5

Any suggestion/hint on how to debug where the error occured?

**Comment by LXGaming** [comment]
I'll update the plugin first as it's a bit out-of-date and see if the issue remains

**Comment by LXGaming** [comment]
I believe this issue is due to the player missing the `location.get.base` permission. Are you using LuckPerms or have any permission plugin installed?

**Comment by Vigilans** [comment]
No, I haven't done any permission management. I see that [velocity docs](https://docs.papermc.io/velocity/built-in-commands#velocity-plugins) mentioned:
```
/velocity plugins
If the user has the velocity.command.plugins permission, they can view all the plugins currently active on the proxy using /velocity plugins, including the name, authors, and version.
```

In game I can only use `/velocity version`:
![image](https://user-images.githubusercontent.com/20227484/224536276-3464422f-4b5f-49ea-a83f-f5a26a54b1fe.png)

So maybe this is because I am lacking the `velocity.command.plugins` permission. If this is the case with `Location`, maybe Location could follow `/velocity`'s behavior to hide `/location` subcommands if no permission is granted?

**Comment by LXGaming** [comment]
Fixed in [v1.4.1](https://github.com/LXGaming/Location/releases/tag/v1.4.1), I've implemented your suggestion of hiding the commands from appearing in the suggestions when permissions are missing.
